src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -39,6 +39,13 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
     const selectedJson = selectedChannel?.rawJson;
     const itemJson = item?.rawJson;
 
+    if (item?.user?.anon_user_info_color_code) {
+      return renderAnonymousAvatar(
+        item?.user?.anon_user_info_color_code,
+        item?.user?.anon_user_info_emoji_code
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code checks if `item?.user?.anon_user_info_color_code` exists and then calls the `renderAnonymousAvatar` function. However, it doesn't check if `item?.user?.anon_user_info_emoji_code` also exists before calling the function. This could potentially lead to an error if `item?.user?.anon_user_info_emoji_code` is undefined or null. I suggest adding a check for `item?.user?.anon_user_info_emoji_code` as well.

```diff
-    if (item?.user?.anon_user_info_color_code) {
+    if (item?.user?.anon_user_info_color_code && item?.user?.anon_user_info_emoji_code) {
       return renderAnonymousAvatar(
         item?.user?.anon_user_info_color_code,
         item?.user?.anon_user_info_emoji_code
       );
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->